src/overlaybd/fs/zfile/crc32/libaccel_config.h

```diff
@@ -0,0 +1,320 @@
+// SPDX-License-Identifier: LGPL-2.1
+/* Copyright(c) 2019 Intel Corporation. All rights reserved. */
+
+#ifndef _LIBACCFG_H_
```

Comment:
This file should be deleted. We should not have this file checked in. Because it is a header file from accel-config library.